examples/reference/panes/Plotly.ipynb

```diff
@@ -226,7 +238,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Plotly Layouts\n",
+    "## Streaming\n",
+    "\n",
+    "You can stream updates by replacing the entire Figure object. To stream efficiently though you should use patching technique described above.\n",
+    "\n",
+    "You can stream periodically using `pn.state.add_periodic_callback`."
```

Comment:
I would have like to add a streaming example using an async generator, but could not because of https://github.com/holoviz/panel/issues/6758.

Reply:
Async Generator Example added